go.mod

```diff
@@ -26,29 +25,32 @@ require (
 	github.com/docker/go-units v0.0.0-20151230175859-0bbddae09c5a // indirect
 	github.com/exoscale/egoscale v0.12.3
 	github.com/google/go-querystring v0.0.0-20140804062624-30f7a39f4a21 // indirect
+	github.com/google/gofuzz v1.2.0 // indirect
 	github.com/gophercloud/gophercloud v0.7.0
 	github.com/gophercloud/utils v0.0.0-20191129022341-463e26ffa30d
 	github.com/gorilla/mux v1.7.3 // indirect
+	github.com/imdario/mergo v0.3.11 // indirect
 	github.com/juju/loggo v0.0.0-20190526231331-6e530bcce5d8 // indirect
-	github.com/mitchellh/mapstructure v0.0.0-20140721150620-740c764bc614 // indirect
-	github.com/onsi/ginkgo v1.9.0 // indirect
-	github.com/onsi/gomega v1.6.0 // indirect
 	github.com/rackspace/gophercloud v0.0.0-20150408191457-ce0f487f6747
 	github.com/samalba/dockerclient v0.0.0-20151231000007-f661dd4754aa
-	github.com/sirupsen/logrus v1.0.4 // indirect
+	github.com/sirupsen/logrus v1.0.4
 	github.com/skarademir/naturalsort v0.0.0-20150715044055-69a5d87bef62
-	github.com/stretchr/testify v1.5.1
+	github.com/stretchr/testify v1.6.1
 	github.com/tent/http-link-go v0.0.0-20130702225549-ac974c61c2f9 // indirect
 	github.com/urfave/cli v1.11.1-0.20151120215642-0302d3914d2a
 	github.com/vmware/govcloudair v0.0.2
 	github.com/vmware/govmomi v0.23.2-0.20201015235820-81318771d0e0
-	golang.org/x/crypto v0.0.0-20200406173513-056763e48d71
-	golang.org/x/net v0.0.0-20200202094626-16171245cfb2
+	golang.org/x/crypto v0.0.0-20201002170205-7f63de1d35b0
+	golang.org/x/net v0.0.0-20201110031124-69a78807bb2b
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
-	golang.org/x/sys v0.0.0-20200212091648-12a6c2dcc1e4
-	google.golang.org/api v0.17.0
+	golang.org/x/sys v0.0.0-20201112073958-5cba982894dd
+	google.golang.org/api v0.20.0
 	gopkg.in/airbrake/gobrake.v2 v2.0.9 // indirect
 	gopkg.in/gemnasium/logrus-airbrake-hook.v2 v2.1.2 // indirect
-	gopkg.in/yaml.v2 v2.2.7
+	gopkg.in/yaml.v2 v2.3.0
```

Comment:
We should use `sigs.k8s.io/yaml`, it adds in more functionality (handling json tags) at a small performance cost.

Reply:
Will do.

Reply:
This also led to editing `drivers/vmwarevsphere/cloudinit.go`.

Reply:
Backtrack: I was thinking this was net-new but its an existing repo. Lets revert and just upgrade the existing library.

Reply:
Done.